### PR TITLE
milestone two -Interface

### PR DESCRIPTION
**Pull request by AshakaE**
- created the basic information displayed to the game players
- created the variables to store user inputs

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Status: Changes Required
Hello Team!
Sorry, but I think that you've misunderstood the goal of this milestone.
In this milestone not everything must work, the point is to make sure that you thought about everything listed below
Basic game flow is prepared: ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/logic.png) hint from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- [x] [OPTIONAL] game asks players for their names
- [x] [REQUIRED] game tells which player turn it is (without computing who the winner is)
- [x]  [REQUIRED] game asks the player to select from available moves
- [x]  [REQUIRED] game informs player if the selected move is invalid (without computing what an invalid move is)
- [x] [REQUIRED] game displays board after the player move (without computing which spots on the board are taken)
- [x] [REQUIRED] game informs player if the selected move is a winning move (without computing the winning move)
- [x] [REQUIRED] game informs player if the selected move is a draw move (without computing draw)
- [x] [REQUIRED] game repeats all actions for the next player's move

As a detail:
- [x] A loop is needed in this game (recommended is while but anything else will do) ([OdinProject hint](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/loop.png))
- [x] Inside the loop, student should call actions repeated for each move
- [x] A loop needs to be terminated at some point (after a winning move or draw to be more specific). 

Kindly make the changes and submit a new code review request.
Thanks
RIndra Josia

File: bin/main.rb [hunk 1]
```diff
@@ -1,3 +1,67 @@
...
+
+  unless board.validate(row, col)
+    puts 'Invalid Position'
+    next
+  end
+
+  board.update_board(row, col, active_player.symbol)
+  board.print_board
+
+  if board_full?
```
Comment (rindrajosia) [inline bin/main.rb]: Please, check this line. It causes an error. I think, you should write: board.board_full?

**Review by rindrajosia — CHANGES_REQUESTED** [review]
**Status: Changes Required**
Hello Team!
Great work on the project so far, all your efforts are visible from the changes made but we have to make a few more changes to get this milestone Approved. Let me break down the changes for you.

_**Required Changes:**_
- [x] Your game accepts any input including invalid inputs.(eg: A10).
- [x] After the first move the game ends.
- [x] [REQUIRED] game repeats all actions for the next player's move.

Keep up with the efforts towards this project, you are almost there.

Rindra Josia.

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇  
### Hi Team ✋🏽 
Nicely done. You have met the minimum requirements for this milestone to be approved.🤝 It's a thumbs up for me. 👍🏽 You can go ahead and merge this PR. 

Good luck on your project and Happy Coding 💻

**Pull request merged**